src/bp/core/routers/bots/content.ts

```diff
@@ -129,6 +134,70 @@ export class ContentRouter extends CustomRouter {
         res.sendStatus(200)
       })
     )
+
+    this.router.get(
+      '/export',
+      this._checkTokenHeader,
+      this._needPermissions('read', 'bot.content'),
+      async (req, res) => {
+        const elements = await this.cms.getAllElements(req.params.botId)
```

Comment:
what if there's a really big load of elements ? should we break down this function un chunks ?

Reply:
I think there's a couple of places we could hit that issue, but it doesn't seems to be a problem at the moment. I'll leave a todo for future improvement